src/sections/Home/So-Special-Section/so-special-style.js

```diff
@@ -39,6 +61,29 @@ const SoSpecialWrapper = styled.div`
             height:100%;
         }    
     }
+    .slick-prev { 
+        left: -2.5rem;
+        position: relative;
+        top: 14.90rem;
+    }
+    .slick-prev:before{	
+        content: "‹";	
+    }
+    .slick-next:before{	
+        content: "›";
+        line-height: 0;
```

Comment:
Needed for `.slick-next:before` as well

Reply:
what is needed here ?

Reply:
The `line-height: 0` property/value.

Reply:
oh I think you mean `.slick-prev:before` here, made changes assuming it 👍